src/viur/core/render/json/user.py

```diff
@@ -32,3 +32,9 @@ def passwdRecoverInfo(self, msg, skel=None, tpl=None, **kwargs):
 
     def passwdRecover(self, *args, **kwargs):
         return self.edit(*args, **kwargs)
+
+    def secound_factor_add(self, otp_uri=None):
+        return json.dumps({"otp_uri": otp_uri})
+
+    def secound_factor_add_success(self,):
```

Comment:
There's a typo in ALL (the old and your) methods, can we rename it this PR. This shouldn't really be breaking because they are only used internally and I think most projects don't subclass them.

```suggestion
    def second_factor_add(self, otp_uri=None):
        return json.dumps({"otp_uri": otp_uri})

    def secund_factor_add_success(self,):
```